api/util/GenerateTokenScript.js

```diff
@@ -0,0 +1,127 @@
+const fs = require('fs');
+const readline = require('readline');
+const { google } = require('googleapis');
+
+const redColor = '\x1b[31m';
```

Comment:
perhaps put these color definitions in [`api/util/constants.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/util/constants.js)? then we can just import them in this script and in the create-user script